javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,65 @@
+const yearOfBirth=1987;
+const yearFuture=2045;
+const age=yearFuture - yearOfBirth;
+console.log("my age is"  + " " +  age + " " +  "for future");
+
+
+
+let dogYearOfBirth=2024;
+let dogYearFuture=2045;
+
+ let shouldShowResultInDogYears=true;
+let dogAgeInHumanYears=dogYearFuture - dogYearOfBirth;
+ let dogAgeInDogYears= dogAgeInHumanYears * 7;
+
+if(shouldShowResultInDogYears){
+    console.log(dogAgeInDogYears);
+}
+else{
+    console.log(dogAgeInHumanYears);
+}
+
+
+
+
+let volumeInMetersPeter = 8 * 10 * 10; 

```

Comment:
Add house data for Peter and Julia, you can do it with array of objects with name, width, depth and ...